src/hotspot/share/prims/jvm.cpp

```diff
@@ -2305,7 +2279,23 @@ JVM_END
 // The function returns a Klass* of the _scratch_class if the verifier
 // was invoked in the middle of the class redefinition.
 // Otherwise it returns its argument value which is the _the_class Klass*.
-// Please, refer to the description in the jvmtiThreadSate.hpp.
+// Please, refer to the description in the jvmtiThreadState.hpp.
+
```

Comment:
Does this "RedefineClasses support" comment still belong here?

Reply:
I think so. The comment in jvmtiThreadState.hpp has details why this is.  We do a mirror switch before verification apparently because of bug 6214132 it says.